src/preferences/dialog/dlgpreflibrary.cpp

```diff
@@ -386,8 +408,37 @@ void DlgPrefLibrary::slotUpdate() {
                     mixxx::library::prefs::kApplyPlayedTrackColorConfigKey,
                     BaseTrackTableModel::kApplyPlayedTrackColorDefault);
     checkbox_played_track_color->setChecked(applyPlayedTrackColor);
+
+#ifdef __STEM__
+    m_stemEnabled = m_pConfig->getValue(kAppEnableStemConfigKey, false);
+    checkBox_enable_stem->setChecked(m_stemEnabled);
+#endif
 }
 
+#ifdef __STEM__
+void DlgPrefLibrary::updateStemFeature(bool enabled) {
+    if (!enabled) {
+        return;
+    }
+    QMessageBox msg;
+    msg.setIcon(QMessageBox::Warning);
+    msg.setWindowTitle(tr("Are you sure?"));
+    msg.setText(QStringLiteral("<p>%1</p><p>%2</p>")
+                    .arg(tr("Stem support is currently under preview and codec "
+                            "inconsistencies are known and could result in "
+                            "seeking imprecision (cues, loop or beat jump) as "
+                            "well as marker loss in future Mixxx version."),
+                            tr("Are you sure you wish to proceed to enable "
+                               "Stem preview?")));
```

Comment:
```suggestion
                    .arg(tr("Stem file support is currently in a preview state."
                            "There are known seeking imprecision (cues, loop or beat jump)"
                            "This leads to audible glitches and out of sync beats."  
                            "All track markers and waveform data might be loss a in future Mixxx version."),
                            tr("Are you sure you wish to proceed to enable "
                               "Stem preview?")));
```

Reply:
> -> we do encourage users to use the stems-features

No, we don't this way. This PR is really a dead end. Let's go for the solution to enable alac stems by default and forward AAC user to alpha.

Reply:
> This leads to audible glitches and out of sync beats."

That is simply not true. Please don't make up issues here. The only problem with AAC is, upon seek, frame position may be off by a few frames (<1ms). 

> This PR is really a dead end

![image](https://github.com/user-attachments/assets/81a0cf75-5f71-4e0c-8652-024f4e11fff5)

Reply:
>> this leads to audible glitches and out of sync beats."

> That is simply not true. Please don't make up issues here. The only problem with AAC is, upon seek, frame position may be off by a few frames (<1ms).

It is unfortunatly. Everytime you pass a hotcue during seady playback a prefetched chiunk of samples is used. If this has only a view samples offset or suffers decoder settling issues we have a rectangular change in the audio, which can be an audible pop or click.

Reply:
Here some old bugs that should not be reintroduced: 
https://github.com/mixxxdj/mixxx/issues/11267
https://github.com/mixxxdj/mixxx/issues/8814